src/redux/dialogsReducer.js

```diff
@@ -27,11 +26,10 @@ const dialogsReducer = (state = initialState, action) => {
       };
 
     case ACTION_TYPES.SEND_MESSAGE:
-      let body = state.newMessageBody;
+      let body = action.newMessageBody;
```

Comment:
Consider using "const" instead of "let" for variables that do not change in the current scope.

Reply:
done